blackbox/docs/release_notes/2.0.3.txt

```diff
@@ -0,0 +1,45 @@
+.. _version_2.0.3:
+
+=============
+Version 2.0.3
+=============
+
+Released on 2017/06/30.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
```

Comment:
line wrapping looks wrong. please make sure that the whole doc is hard wrapped to 79 chars